### DISCUSSION TITLE
Steps or examples to configure and deploy Django serverless apps

### DISCUSSION THREAD

**Discussion by dominic248** [vercel/vercel#4890] [Help]
The documentation or examples doesn't contain info on how to configure and deploy Django serverless apps on Vercel.
Any steps/examples available on how to configure and deploy Django serverless apps on Vercel?

**Comment by MichelML** [comment]
+1

**Comment by advl** [comment]
It works out of the box. Just serve your asgi/wsgi in the api directory.

If you rely on pgsql note that you should use psycopg2-binary instead of psycopg2, as psycopg2 requires some files that are not available in the build runtime
